modules/RTC/TraceablePeerConnection.js

```diff
@@ -1460,6 +1512,44 @@ TraceablePeerConnection.prototype._getSSRC = function(rtcId) {
     return this.localSSRCs.get(rtcId);
 };
 
+/**
+ * Munges the order of the codecs in the SDP passed based on the preference
+ * set through config.js settings. All instances of the specified codec are
+ * moved up to the top of the list when it is preferred. The specified codec
+ * is deleted from the list if the configuration specifies that the codec be
+ * disabled.
+ * @param {RTCSessionDescription} description that needs to be munged.
+ * @returns {RTCSessionDescription} the munged description.
+ */
+TraceablePeerConnection.prototype._mungeCodecOrder = function(description) {
+    if (!(this.codecPreference && this.codecPreference.mimeType) || browser.supportsCodecPreferences()) {
```

Comment:
The check for `this.codecPreference` should be enough, given that it's initialized only for correct values now